AntennaTracker/Tracker.h

```diff
@@ -278,6 +290,11 @@ class Tracker : public AP_HAL::HAL::Callbacks {
     void tracking_manual_control(const mavlink_manual_control_t &msg);
     void update_armed_disarmed();
 
+    // Mission library
+    AP_Mission mission{
```

Comment:
Do we need the mission object for anything?  we don't support missions.. is it required in order to hold the home position?

Reply:
its just so we can use the GCS mission messages, the mavlink gcs common returns errors to mission planner when it tries to read way point zero on arming. The fix is to construct the mission library so its not nulptr, the other way to fix is add custom mavlink messages for all the mission messages that know not to look for the mission library.

Reply:
Is it worthwhile to change AP_Mission to check for nullptr callbacks?

Reply:
Yup, sounded like such a good idea that I went ahead and did it. See https://github.com/ArduPilot/ardupilot/pull/10542

Using that, you probably don't need to create a mission instance. if you did, you could just pass it nullptr functions instead of these empty ones.